docs/root/version_history/current.rst

```diff
@@ -51,7 +50,7 @@ Minor Behavior Changes
 * router: now consumes all retry related headers to prevent them from being propagated to the upstream. This behavior may be reverted by setting runtime feature `envoy.reloadable_features.consume_all_retry_headers` to false.
 * stats: the fake symbol table implemention has been removed from the binary, and the option "--use-fake-symbol-table" is now a no-op with a warning.
```

Comment:
render as :option:`--use-fake-symbol-table` (and --enable-fine-grain-logging) for consistency?

Reply:
ah, turns out I had to revert this one - as a "not real flag" it's not documented, so flagging it as an option causes problems.